straxen/rundb.py

```diff
@@ -74,16 +65,7 @@ def __init__(self,
 
         self.hostname = socket.getfqdn()
 
-        if mongo_url is None:
-            mongo_url = get_mongo_url(self.hostname)
-
-        self.client = pymongo.MongoClient(mongo_url)
-
-        if mongo_dbname is None:
-            mongo_dbname = default_mongo_dbname
-        if mongo_collname is None:
-            mongo_collname = default_mongo_collname
-        self.collection = self.client[mongo_dbname][mongo_collname]
```

Comment:
@ershockley why delete this? This is not a good idea as one cannot intuitively change which user/collection one is using. E.g. on the eventbuilder we have a production and testing mode where we use different access to the database for obvious reasons.

Reply:
Also testing with another database becomes quite inflexible